### PR TITLE
First cut at adding ".sth" and ".sth-consistency" docs for DNS.

### PR DESCRIPTION
**Pull request by aeijdenberg**
@pphaneuf  Please take a look.

### CODE REVIEW

**Comment by aeijdenberg** [comment]
@eranmes since @pphaneuf has gone quiet...

File: dns/draft-ct-over-dns.md [hunk 1]
```diff
@@ -56,9 +56,109 @@ In this manner only a proxy for aggregate usage of sites contained in a log will
...
+Inputs:
+:
+: `domain_for_log` is a value stored in log metadata.  For example, a name server operated by Google for the Pilot log may have the domain `pilot.ct.googleapis.com`.
+
+Outputs:
+:
+: `tree_size` The size of the tree, in entries, in decimal.
+: `timestamp` The timestamp, in decimal.
+: `sha256_root_hash` The Merkle Tree Hash of the tree, in base64.
+: `tree_head_signature` A TreeHeadSignature for the above data.
```
Comment (pphaneuf) [inline dns/draft-ct-over-dns.md]: The formatting for this comes out a bit weird on GitHub, that's expected?

File: dns/draft-ct-over-dns.md [hunk 2]
```diff
@@ -56,9 +56,109 @@ In this manner only a proxy for aggregate usage of sites contained in a log will
...
+
+And receive the following response:
+
+               +---------------------------------------------------+
+    Header     | OPCODE=SQUERY, RESPONSE, AA                       |
+               +---------------------------------------------------+
+    Question   | QNAME=D4<snip>.com., QCLASS=IN, QTYPE=TXT         |
+               +---------------------------------------------------+
+    Answer     | QNAME=D4<snip>.com., 604800 IN TXT 123456.1447255 |
+               | 802227.aO4z<snip>U=.BAMAR<snip>Vl                 |
```
Comment (pphaneuf) [inline dns/draft-ct-over-dns.md]: BAMAR? Is that a real signature?!? :smiley:

File: dns/draft-ct-over-dns.md (hunk 1 above)
Comment (pphaneuf) [inline dns/draft-ct-over-dns.md]: Yeah, no worries, just making sure this is as expected.

**Comment by pphaneuf** [comment]
LGTM

**Pull request merged**